Tests/MongoSwiftTests/ReadConcernTests.swift

```diff
@@ -202,4 +177,23 @@ final class ReadConcernTests: MongoSwiftTestCase {
             try checkReadConcern(coll5, majority, "collection retrieved with majority RC from \(dbDesc)")
         }
     }
+
+    func testRoundTripThroughLibmongoc() throws {
+        let rcs: [ReadConcern] = [
+            ReadConcern(),
+            ReadConcern(.local),
```

Comment:
writing this out made me think we should switch to a similar API to the one we have now for `ReadPreference`. basically static properties corresponding to each `ReadConcern` mode, plus a static method for creating one with an unknown level for forward compatibility. thoughts?

Reply:
SGTM, you could probably also do it for `WriteConcern` for some expected common values (e.g. `majority`)

Reply:
cool, I opened [SWIFT-815](https://jira.mongodb.org/browse/SWIFT-815) to track that work